net/oui/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+revision            0
+
+version             0.1.8
+name                oui
+
+homepage            https://github.com/thatmattlove/oui
+
+categories          net
+
+description         MAC Address CLI Toolkit
+
+long_description    Look up MAC Address vendors and convert MAC Addresses \
+                    to other formats — offline and at the console.
+
+maintainers         {stunninglyclear.com:matt @thatmattlove} openmaintainer
+
+license             BSD-3-Clause-Clear
```

Comment:
This is not a license name we recognize. (We do not use SPDX license identifiers yet.) We use the license name "BSD" for this.